stream-chat-android-client/src/main/java/io/getstream/chat/android/client/api/models/QueryChannelRequest.kt

```diff
@@ -93,6 +93,10 @@ public open class QueryChannelRequest : ChannelRequest<QueryChannelRequest> {
         return keys.contains(Pagination.AROUND_ID.toString())
     }
 
+    public fun isFilteringMessages(): Boolean {
+        return messages.keys.any { key -> Pagination.values().map { it.toString() }.contains(key) }
```

Comment:
What do you think about replacing it with:
```Pagination.values().map { it.toString() }.fold(false) { currentValue, paginationKey -> currentValue || messages.keys.contains(paginationKey) }```?
It's not a big performance improvement but we avoid iterating through `messages.keys` and `Pagination.values()` mapping on every iteration

Reply:
Yes, this works.

Reply:
Could we also try to intersect the two sets of keys and then see if the intersection isn't empty?

Reply:
good idea!